targets/audio

```diff
@@ -3,15 +3,19 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 REQUIRES='core'
-DESCRIPTION='Support audio playback via Chromium OS'\''s audio system.'
+DESCRIPTION='Support audio playback via Chromium OS audio system (CRAS).'
```

Comment:
Acronyms are hard and I always forget what they actually mean. This was more of a "hey, remember this", I don't really need this to go in, haha.